jsapp/js/actions.es6

```diff
@@ -400,31 +400,31 @@ actions.resources.deployAsset.failed.listen(function(data, dialog_or_alert){
     // setContent() removes the input box, but the value is retained
     var msg;
     if (data.status == 500 && data.responseJSON && data.responseJSON.error) {
-      msg = `<code><pre>${data.responseJSON.error}</pre></code>`;
+      msg = `<pre>${data.responseJSON.error}</pre>`;
     } else if (data.status == 500 && data.responseText) {
-      msg = `<code><pre>${data.responseText}</pre></code>`;
+      msg = `<pre>${data.responseText}</pre>`;
     } else {
-      msg = t('please check your connection and try again.');
+      msg = t('Please check your connection and try again.');
```

Comment:
Ah, I wasn't aware of the djangojs.po process. I can revert this (and a few other similar changes) and make them in the djangojs.po file.